intelmq/bots/parsers/zoneh/parser.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+ZoneH CSV defacement report parser
+"""
+import csv
+import io
+from urllib.parse import urlparse
+
+from intelmq.lib import utils
+from intelmq.lib.bot import ParserBot, RewindableFileHandle
+from intelmq.lib.message import Event
+
+
+class ZoneHParserBot(ParserBot):
```

Comment:
You are using the ParserBot here and below you are overwriting it's `process` method. So you can use the plain `Bot` here.